syntax = "proto3";

package router_serialize;

message RouterSettings {
	int32 bus_wait_time = 1;
	int32 bus_velocity = 2;
}